packages/cashier/src/pages/on-ramp/on-ramp.tsx

```diff
@@ -8,8 +7,29 @@ import CashierLocked from 'Components/cashier-locked';
 import OnRampProviderCard from './on-ramp-provider-card';
 import OnRampProviderPopup from './on-ramp-provider-popup';
 import SideNote from 'Components/side-note';
+import RootStore from 'Stores/types';
 import './on-ramp.scss';
 
+type TOnRampProps = {
+    filtered_onramp_providers: any[];
```

Comment:
Don't we know the type?
Or could you add a `TODO` comment to replace `any` with the proper type when we refactor the onramp store as well?